CHANGELOG.md

```diff
@@ -95,6 +95,11 @@
   - `SummaryDataPoint.Flags` -> `SummaryDataPoint.FlagsImmutable`
   - `MetricDataPointFlags` -> `MetricDataPointFlagsImmutable`
   - `NewMetricDataPointFlags` -> `MetricDataPointFlagsImmutable`
+- Deprecate `ptrace.TraceState` in favor of `pcommon.TraceState`. (#6052)
+  - `ptrace.Span.TraceState` in favor of `ptrace.Span.TraceStateStruct().AsRaw()`
+  - `ptrace.Span.SetTraceState` in favor of `ptrace.Span.TraceStateStruct().FromRaw`
+  - `ptrace.SpanLink.TraceState` in favor of `ptrace.SpanLink.TraceStateStruct().AsRaw()`
+  - `ptrace.SpanLink.SetTraceState` in favor of `ptrace.SpanLink.TraceStateStruct().FromRaw`
```

Comment:
Let's maybe clarify here that `TraceStateStruct` is a temporary name that will be replaced back to `TraceState` in the next release?

Reply:
Done.